6/7 release draft (#282)
gridai/grid-docs: changelog/2022-06-07.md

## :zap: June 7, 2022

**CLI version: 0.8.58** 

> Today's release includes several bug fixes to improve the overall experience with Grid. 

Reviewer: ```suggestion
Today's release includes several bug fixes that improve the Grid experience.
```
